src/main/java/uk/ac/ebi/ena/readtools/loader/fastq/DataSpot.java

```diff
@@ -522,4 +525,19 @@
             .append( null == quals ? "null" : quals.length() )
             .toString();
     }
+
+    private void handleInvalidBases(String bases) {
+        Matcher matcher = p_bases.matcher("");
+
+        //remove the trailing '+' symbol.
+        bases = bases.substring(0, bases.length() - 1);

```

Comment:
It might be better to check if the last character is '+' and if it is then remove it. Logically the '+' should not be included in the bases. It is a separator between bases and qualities.

Reply:
yes. it is handled properly at a later point.